Overview.src.html

```diff
@@ -381,9 +381,9 @@ <h4 id=terminology-headers>Headers</h4>
  <li>`<code title>Set-Cookie2</code>`
 </ul>
 
-<p>A <dfn>CORS-safelisted response-header name</dfn>, given a
-<span title=concept-response-header-list>header list</span> <var>list</var>, is a
-<span title=concept-header>header</span> <span title=concept-header-name>name</span> that is one of:
+<p>A <dfn id="cors-safelisted-response-header-name">CORS-safelisted response-header name</dfn>, given a
+<span title=concept-response-cors-exposed-headers-list>CORS-exposed headers list</span> <var>list</var>, is a
+<a href="#concept-header" title="concept-header">header</a> <a href="#concept-header-name" title="concept-header-name">name</a> that is one of:
```

Comment:
This also looks wrong. It seems like you copied output from `Overview.html` and put it in `Overview.src.html`. Did Anolis fail for some reason? If you cannot use Anolis, please only submit a patch to `Overview.src.html` and leave `Overview.html` to me. See the README for instructions.